target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8068-cryptid-common.dtsi

```diff
@@ -0,0 +1,226 @@
+#include "qcom-ipq8064-v2.0.dtsi"
```

Comment:
Please add an SPDX license identifier.

Reply:
I am just about to push some changes, including moving the tlc591xx package/kernel/modules/leds and fixing some whitespace issues. Will add this now then push.